ui/src/app/icons/card-menu-icon.tsx

```diff
@@ -1,16 +0,0 @@
-import * as React from 'react';
-
-
-export const CardMenuIconComponentReact = () => <svg
-  version='1.1'
-  viewBox='0 0 18 18'
-  preserveAspectRatio='xMidYMid meet'
-  xmlns='http://www.w3.org/2000/svg'>
-  <g fill='currentColor' fillRule='evenodd'>
```

Comment:
is this used anywhere?

Reply:
I did not find any references to this icon in the code.

Reply:
confirmed 👍